client/src/index.js

```diff
@@ -1,17 +1,19 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
```

Comment:
You deleted this file, didn't you? If so, please remove the import!

Reply:
Done!